<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>init</class>
 <widget class="QWidget" name="init">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="inputFname">
     <property name="text">
      <string>Choose Input file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="inputDicomHDR">
     <property name="text">
      <string>Choose dicom Header</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="proxyRender" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="headerEntries">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputFname</sender>
   <signal>clicked()</signal>
   <receiver>init</receiver>
   <slot>loadInImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>init</receiver>
   <slot>OnSliderChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputDicomHDR</sender>
   <signal>clicked()</signal>
   <receiver>init</receiver>
   <slot>loadIndcmHDR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadInImage()</slot>
  <slot>OnSliderChange()</slot>
  <slot>loadIndcmHDR()</slot>
 </slots>
</ui>
